src/lib/axios.js

```diff
@@ -24,5 +26,9 @@ module.exports = function (req, res, next) {
     req.axios.defaults.headers.common["x-scenario-id"] = req.scenarioIDHeader;
   }
 
+  if (req?.headers["forwarded"] && req.axios?.defaults?.headers?.common) {
+    req.axios.defaults.headers.common["x-forwarded-for"] = userIpAddress(req);
```

Comment:
We should not pass the `req` object into the userIpAddress for 2 reasons

1. The `headers["forwarded"]` usage is duplicated in `axios.js` and in `user-ip-address`. If that header changed, we need to change both places.
2. The `user-ip-address` function is returning a string based upon a single header value - it does not need all of the headers, it certainly does not need the entire request object. If `user-ip-address` happened to accidentally change something in request, that would affect every single web request. It is safer to avoid all that, and pass in a header string value.